src/platforms/flutter/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -165,3 +165,43 @@ Future<void> main() async {
 ```
 
 Slow and frozen frames are Mobile Vitals, which you can learn about in the [full documentation](/product/performance/mobile-vitals).
+
+### AssetBundle Instrumentation
+
+The [AssetBundle](https://api.flutter.dev/flutter/services/AssetBundle-class.html) Instrumentation provides insight into how long your App takes to load its assets, e.g. files.
```

Comment:
```suggestion
The [AssetBundle](https://api.flutter.dev/flutter/services/AssetBundle-class.html) instrumentation provides insight into how long your app takes to load its assets, such as files. It can be enabled by wrapping the `runApp` method with a `DefaultAssetBundle` and `SentryAssetBundle`.
```